docs/feature-hermetic-builds.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+title: Hermetic builds with Mock
+---
+
+Hermetic builds with Mock
+=========================
+
+Mock (v5.7+) supports hermetic RPM builds, sometimes referred to as "isolated"
+or "offline" builds.  For more details, see the
+[SLSA "hermetic" definition][SLSA future].
+
+Quick start
+-----------
+
+For the impatient, the TL;DR steps are as follows:
+
+    # we want to build this package
+    srpm=your-package.src.rpm
+
+    # we'll create a local repository with pre-fetched RPMs/bootstrap
+    repo=/tmp/local-repo
+
+    # resolve build deps for the given SRPM, in this case for Fedora Rawhide
+    mock --calculate-build-dependencies -r fedora-rawhide-x86_64 "$srpm"
+
+    # find the lockfile in Mock's resultdir
+    lockfile=/var/lib/mock/fedora-rawhide-x86_64/result/buildroot_lock.json
+
+    # create a local RPM repository (+ download bootstrap image)
+    mock-hermetic-repo --lockfile "$lockfile" --output-repo "$repo"
+
+    # perform the hermetic build!
+    mock --hermetic-build "$lockfile" "$repo" "$srpm"
+
+What an "hermetic build" is..
+-----------------------------
+
+The term "isolated build" is often used in different contexts within
+Mock's terminology.  Historically, when we said that "Mock isolates the build,"
+we typically meant that Mock creates a *buildroot* (also referred to as a *build
+directory* or *build chroot*) and runs the (Turing-complete, and thus
+potentially insecure) *RPM build* process (i.e., a call to `/usr/bin/rpmbuild`)
```

Comment:
Instead of "Turing-complete, and thus potentially insecure" I would say "with some steps executed under root user".

Reply:
But this is was there even previously. So I will not block a merge on this.